src/Microsoft.VisualStudio.ProjectSystem.FSharp/ProjectSystem/LanguageServices/FSharpParseBuildOptions.cs

```diff
@@ -2,19 +2,56 @@
 
 using System.Collections.Generic;
 using System.ComponentModel.Composition;
-using Microsoft.CodeAnalysis.CSharp;
+using System.IO;
+using Microsoft.CodeAnalysis;
 
 namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices
 {
     [Export(typeof(IParseBuildOptions))]
     [AppliesTo(ProjectCapability.FSharp)]
     internal class FSharpParseBuildOptions : IParseBuildOptions
     {
+        private const string ReferencePrefix = "-r:";
```

Comment:
There is a `--reference` option, but that's never exposed via the design time build; only `-r` is used.  I'll be sure to add `--reference` support to the official parser that will live in the F# code.